sdk/search/Azure.Search.Documents/tests/DocumentOperations/GetDocumentTests.cs

```diff
@@ -304,6 +305,7 @@ public async Task CannotAlwaysDetermineCorrectType()
                 ["hotelId"] = "1",
                 ["hotelName"] = "2015-02-11T12:58:00Z",
                 ["location"] = TestExtensions.CreateDynamicPoint(-73.975403, 40.760586), // Test that we don't confuse Geo-JSON & complex types.
+                ["geoLocation"] = TestExtensions.CreateDynamicGeoPoint(-73.975403, 40.760586), // Test that we don't confuse Geo-JSON & complex types.
```

Comment:
Nit: if this corresponds to Microsoft.Spatial, the "geo" prefix seems a bit confusion. Maybe something like "ms" as a prefix or "spatial" or something that indicative?

Reply:
...at least, in this point of the PR I'm assuming this was to effectively duplicate geo properties to do both GeoJson and Microsoft.Spatial.

Reply:
The new one property 'geoLocation' is a `GeoJson` type. The existing one - 'location' is the MS.Spatial type.